sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Constants.java

```diff
@@ -50,4 +50,13 @@ public static synchronized void handleApiCallFailure(Exception e) {
             }
         }
     }
+    /**
+     * Cosmosdb token cache file
+     */
+    static final String TOKEN_CACHE_FILE = "token_cache";
+
+    static final String PARTITION_NAMES = "partition_names";
+
```

Comment:
No empty line here.

Reply:
Looks like there's still 2 blank lines here (57-58), should only be 1

Reply:
weird, this is not the file looks like in my local...